### PR TITLE
metrics,store/tikv: add a duration metric for tikv batch client unavailable

### PR DESCRIPTION
**Pull request by tiancaiamao**
### What problem does this PR solve? 

When batch client is unavailable, TiDB can't comminute with TiKV
I think it's necessary to have a metric for it

### What is changed and how it works?

Add metric

### Check List 

Tests 

 - No code

### CODE REVIEW

**Comment by tiancaiamao** [comment]
PTAL @hicqu @jackysp

File: store/tikv/client.go [hunk 1]
```diff
@@ -152,6 +153,7 @@ func (c *batchCommandsClient) batchRecvLoop(cfg config.TiKVClient) {
 				// TODO: Use a more smart backoff strategy.
 				time.Sleep(time.Second)
 			}
+			metrics.TiKVBatchClientUnavailable.Observe(time.Since(now).Seconds())
```
Comment (shenli) [inline store/tikv/client.go]: If the previous loop cannot stop, you will never get here.

Comment (tiancaiamao) [reply store/tikv/client.go]: If it never break the loop, tidb is unavailable... we'll know that by other metrics such as QPS may drop to zero @shenli

**Pull request merged**